frontend/css/app.scss

```diff
@@ -557,6 +557,126 @@
     }
 }
 
+#overlay {
+    position: fixed;
+    left: 50%;
+    top: 50%;
+    background-color: #ffffff;
+    width: 400px;
+    height: 550px;
+    margin-left: -200px;
+    margin-top: -275px;
+    z-index: 120;
+
+    .dark{
+        padding-left:20px;
+        padding-right:20px;
+    }
+
+    .links{
+        margin-top: 20px;
+        text-align: center;
+        padding: 0 56px;
+
+        .btn{
+            display: block;
+            margin-bottom: 8px;
+
+        }
+    }
+
+    .close{
+        position: absolute;
+        right: 10px;
+        top: 2px;
+
+        i{
+            cursor: pointer;
+        }
+    }
+
+    input:not([type="submit"]){
+        display: block;
+        width: 100%;
+        margin-bottom: 20px;
+        border: none;
+        border-bottom: 2px solid white;
+        background-color: transparent;
+        font-size: $fontSizeLarge;
+        color: white;
+        text-align: center;
+
+        &:focus{
+            outline: none;
+        }
+
+        &.errorField{
+            border-bottom-color: $colorErrorBackground;
+            color: $colorErrorBackground;
+        }
+    }
+
+    input[type="submit"] {
+        margin-top: 10px;
+    }
+
+    input[type="checkbox"] {
+        display: none;
+    }
+
+    label{
+        cursor: pointer;
+    }
+
+    input[type="checkbox"] + label::before {
+        content: ' ';
+        display: inline-block;
+        width: 16px;
+        height: 16px;
+        border: 1px solid $colorBackground;
+        border-radius: 50%;
+        margin-right: 10px;
+        background-color: lighten($colorBackGroundInvertedLight, 10%);
+    }
+
+    input[type="checkbox"]:checked + label::before {
+        background-color: $colorBackground;
+    }
+
+    .error input:not([type="submit"]){
+        border-bottom-color: $colorErrorBackground;
+        color: $colorErrorBackground;
+    }
+
+    .msg{
+        padding: 4px;
+        font-size: $fontSizeNormal + 4;
+        font-weight: bold;
+
+        &.error{
+            background-color: $colorErrorBackground;
+            color: white;
+        }
+
+        &.success{
+            background-color: $colorSuccess;
+            color: white;
+        }
+    }
+}
+#fade {
```

Comment:
Can be :-) But I assume that there will be only one modal window, so it should not matter that much... Do you want me to still change it?